src/main/java/org/triplea/client/launch/LaunchScreen.java

```diff
@@ -0,0 +1,654 @@
+package org.triplea.client.launch;
```

Comment:
I was looking at `org.triplea.client.ui.javafx` earlier and was not necessarily sure about the package placement.

`org.triplea.client.ui.swing.launch` is a logical suggestion, though a problem I see is that `launch` is pushed below `ui` and is pushed below even `swing`. The `launch` window is *not* a 'child' or a  detail of `swing` :)   

This may be the time when we start naming the clients we are building, `ui` I think is the folder level where things went a bit wrong. That probably should be the client "name".